src/character.cpp

```diff
@@ -8013,52 +8013,79 @@ float Character::get_bmi() const
 }
 
 std::string Character::get_weight_string() const
+{
+    std::pair<std::string, nc_color> weight_pair = get_weight_description();
+    return colorize( weight_pair.first, weight_pair.second );
+}
+
+std::pair<std::string, nc_color> Character::get_weight_description() const
 {
     const float bmi = get_bmi();
+    std::string weight_string;
+    nc_color weight_color = c_light_gray;
     if( get_option<bool>( "CRAZY" ) ) {
         if( bmi > character_weight_category::morbidly_obese + 10.0f ) {
-            return colorize( _( "AW HELL NAH" ), c_red );
+            weight_string = "AW HELL NAH";
```

Comment:
I recommend you to bracket these strings to be shown by `translate_marker` like
```c++
weight_string = translate_marker( "AM HELL NAH" );
```
The dummy macro does nothing in code, but serves as a marker for `xgettext` to add it for translation.

Reply:
Done now, thank you for reminding me to think of translations.